internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -263,10 +249,6 @@ func (s *BackendService) Parameters() ([]*cloudformation.Parameter, error) {
 				ParameterKey:   aws.String(WorkloadRulePathParamKey),
 				ParameterValue: s.manifest.RoutingRule.Path,
 			},
-			{
-				ParameterKey:   aws.String(WorkloadRulePathSliceParamKey),
-				ParameterValue: aws.String(strings.Join([]string{aws.StringValue(s.manifest.RoutingRule.Path)}, ",")),
-			},
 			{
 				ParameterKey:   aws.String(WorkloadStickinessParamKey),
```

Comment:
I want to make sure: I guess `s.manifest.RoutingRule.IsEmpty()` is equivalent to `.ALB` in go template?

Reply:
yes, you are right.